packages/dhis2/src/Utils.js

```diff
@@ -40,9 +45,13 @@ export function selectId(resourceType) {
 
 // Write a unit test for this one
```

Comment:
Let's remove this comment.

I don't really think `handleResponse` should have its own unit test at this point.